synapseclient/annotations.py

```diff
@@ -150,9 +150,13 @@ def to_submission_status_annotations(annotations, is_private=True):
     Example: Using this function
         Adding and converting annotations
 
-            from synapseclient.annotations import to_submission_status_annotations, from_submission_status_annotations
+            from synapseclient.client import Synapse
+            from synapseclient.annotations import to_submission_status_annotations
             from datetime import datetime as Datetime
 
+            ## Initialize a Synapse object
+            syn = Synapse()
```

Comment:
Usually we've been adding a line with this as well:
```
syn.login()
```

Just to drill in with repetition that actions require you to intentionally log into the system. We have a small few things anonymous people can do, but this does not look like one of those things.